src/kOS/Module/kOSProcessor.cs

```diff
@@ -82,7 +95,7 @@ public void Activate()
             OpenWindow();
         }
 
-        [KSPField(isPersistant = true, guiName = "Required Power", guiActive = true)]
+        [KSPField(isPersistant = true, guiName = "kOS Required Power", guiActive = true, guiActiveEditor = true)]
```

Comment:
Actually, as written now:
```
RequiredPower = (float)(volumePower + SafeHouse.Config.InstructionsPerUpdate * ECPerInstruction / TimeWarp.fixedDeltaTime);
```
So, when the game is actually running, it is the real current power requirement.  When in the editor, it is "max" power (based on current IPU), but in flight it is the real value.  You may also notice that I made it visible in the editor, so when you change the "max ipu" setting (which is not yet obsolete) or the disk size you can see the effect.  I expect that we will set the IPU per cpu using a tweakable as well, so the ability to observe how your changes effect power draw may be very useful, especially when designing RemoteTech satellites.